x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial-grid.csv-spec

```diff
@@ -290,27 +376,43 @@ count:long | centroid:geo_point
 # Tests for geo_grid function: ST_GEOTILE
 ###############################################
 
-geotileStringToLong
-required_capability: spatial_grid
+toGeotile
+required_capability: spatial_grid_types
 
-// tag::geotile_to_long[]
-ROW geotile = "4/8/5"
-| EVAL geotileLong = ST_GEOTILE_TO_LONG(geotile)
-// end::geotile_to_long[]
+// tag::to_geotile[]
+ROW string = "4/8/5"
+| EVAL geotile = TO_GEOTILE(string)
+// end::to_geotile[]
 ;
 
-// tag::geotile_to_long-result[]
-geotile:keyword | geotileLong:long
-4/8/5           | 1152921508901814277
-// end::geotile_to_long-result[]
+// tag::to_geotile-result[]
+string:keyword | geotile:geotile
+4/8/5          | 4/8/5
+// end::to_geotile-result[]
+;
+
+toGeotileToLong
+required_capability: spatial_grid_types
+
+// tag::to_geotile_to_long[]
+ROW string = "4/8/5"
+| EVAL geotile = TO_GEOTILE(string)
+| EVAL geotileLong = TO_LONG(geotile)
```

Comment:
naive question: why do we allow turning geotiles and hashes into longs? Is this conversion canonical and can be used by the user?

I'm wondering if we're unnecessarily exposing an implementation detail here.

Reply:
Thanks! Makes sense to me.